docs/troubleshooting.md

```diff
@@ -0,0 +1,25 @@
+# Troubleshooting
+
+This is a place to record workarounds to some common problems.
+
+## User didn't receive password reset email
+
+Signon currently uses Amazon Simple Email Service (SES) to send emails to users. Amazon SES emails don’t pass strict DMARC setups. This means that some mail servers block Signon emails because they can't tell them apart from phishing or spoof emails.
```

Comment:
It's probably worth adding a link to either https://dmarc.org/ or https://en.wikipedia.org/wiki/DMARC.
```suggestion
Signon currently uses Amazon Simple Email Service (SES) to send emails to users. Amazon SES emails don’t pass strict [DMARC](https://dmarc.org/) setups. This means that some mail servers block Signon emails because they can't tell them apart from phishing or spoof emails.
```